remsfal-services/remsfal-platform/src/main/java/de/remsfal/service/control/NotificationController.java

```diff
@@ -1,23 +1,26 @@
 package de.remsfal.service.control;
 
+import de.remsfal.core.model.CustomerModel;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
-
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Emitter;
 import org.jboss.logging.Logger;
+import de.remsfal.core.json.UserJson;
 
-import de.remsfal.core.model.UserModel;
-
-/**
- * @author Alexander Stanik [alexander.stanik@htw-berlin.de]
- */
 @ApplicationScoped
 public class NotificationController {
-    
+
     @Inject
     Logger logger;
-    
-    public void informUserAboutProjectMembership(final UserModel user) {
-        logger.infov("TODO: User {0} will be informed about project membership", user.getEmail());
-    }
 
-}
+    @Inject
+    @Channel("user-notification")
+    Emitter<UserJson> notificationEmitter;
+
+    public void informUserAboutProjectMembership(final CustomerModel user) {
+        UserJson json = UserJson.valueOf(user);
+        logger.infov("Sending user-notification for {0}", json.getEmail());
+        notificationEmitter.send(json);
```

Comment:
Please write an integration test to check the kafka message function. See https://quarkus.io/guides/kafka#testing-using-a-kafka-broker